vtm-jts/src/org/oscim/layers/vector/geometries/Style.java

```diff
@@ -259,6 +262,12 @@ public Builder pointReduction(boolean pointReduction) {
             return this;
         }
 
+        public Builder pointReduction(float dropDistance) {
+            this.pointReduction = false;
+            this.dropDistance = dropDistance;
+            return this;
+        }
```

Comment:
For clarity we should avoid changing multiple parameters in one setter.

As we add the `dropDistance` parameter that already exists in `LineBucket`,
we can remove the `pointReduction` and simplify the functions.

Reply:
I hesitated to remove the "pointReduction" parameter because I feared that change would be breaking to existing users of the lib. Is this not an issue?

Reply:
This is not a common method and we will mention it in changelog.

Please see commit https://github.com/mapsforge/vtm/commit/ee602a39d466fd3ed80c96fbb3a543712060ce83, something like that should work?